### ISSUE TITLE
Multi-language model

### ISSUE THREAD

**Issue by WoBuChiTang** [jasonppy/VoiceCraft#52]
Hi, thank you for your excellent repo!
Do you have any plan to develop a multi-language model?

**Comment by NLPpupil** [comment]
同问，不知道中文效果怎么样？

**Comment by xeeroxxx** [comment]
> 同问，不知道中文效果怎么样？

I don't think so. The currently released models work very well with English. However, I've tried and they can't recognize Chinese characters. I'm not sure if I'm doing something wrong.

I'm not sure if they will release models for Chinese, or if we can train them ourselves. Don't know much about technical training stuff.
